R/module_filter_manager.R

```diff
@@ -147,10 +107,12 @@ filter_manager_srv <- function(id, filtered_data_list, filter) {
       )
     })
 
-    # Call snapshot manager.
-    snapshot_manager_srv("snapshot_manager", slices_global, mapping_matrix, filtered_data_list)
-
-    modules_out # returned for testing purpose
+    list(
+      slices_global = slices_global,
+      mapping_matrix = mapping_matrix,
+      filtered_data_list = filtered_data_list,
```

Comment:
I think this name is not precise enough. Maybe `modules_filtered_data` or `flatten_filtered_data`? When looking at this object name in other modules it might be difficult to grasp it in the first time

Reply:
I was going to suggest a more fitting name after other changes are introduced. 
This is why it's still a draft :smiling_imp:

Reply:
I renamed objects and utility functions throughout the managers.